src/devana/syntax_abstraction/usingnamespace.py

```diff
@@ -39,6 +40,16 @@ def from_namespace(cls, namespace: str, parent: Optional = None) -> "UsingNamesp
         result._namespace = namespace
         return result
 
+    @classmethod
+    @init_params(skip={"cls", "parent"})
+    def from_params( # pylint: disable=unused-argument
+            cls,
+            parent: Optional = None,
+            namespaces: Optional = None,
+            lexicon: Optional = None
```

Comment:
Why isn't this typed?

Reply:
cuz im lazy

Reply:
I've improved it, but I need to look at it again more carefully, because in some places it takes a few dodgy types